resolvers/deno/cjs.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 
-use std::sync::Arc;
-
+#[allow(clippy::disallowed_types)]
+use crate::sync::MaybeArc;
```

Comment:
We should refactor this to not require adding disallowed_types. In the other crates, we create types like `InNpmPackageCheckerRc` and then only use the clippy allow on there.